src/miniscript/hash256.rs

```diff
@@ -0,0 +1,94 @@
+// Miniscript
+// Written in 2022 by
+//     Sanket Kanjalkar <sanket1729@gmail.com>
+//
+// To the extent possible under law, the author(s) have dedicated all
+// copyright and related and neighboring rights to this software to
+// the public domain worldwide. This software is distributed without
+// any warranty.
+//
+// You should have received a copy of the CC0 Public Domain Dedication
+// along with this software.
+// If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+//
+
+//! Hash256 type
+//!
+//! Functionality similar to `sha256d` Hash type in bitcoin, but the FromStr
+//! `Display` are *NOT* REVERSE.
```

Comment:
```suggestion
//! This module is _identical_ in functionality to the `bitcoin_hashes::sha256d` hash type
//! but the `FromHex` and `ToHex` implementations use `DISPLAY_BACKWARDS = false`.
```

Reply:
Changed and also added a comment about `FromStr` and `Display`